tests/system/src/test/java/test/javafx/embed/swing/SwingNodeMemoryLeakTest.java

```diff
@@ -69,18 +72,15 @@ public static void teardownOnce() {
     }
 
     @Test
-    public void testSwingNodeMemoryLeak() {
-        if (PlatformUtil.isMac()) {
-            assumeTrue(Boolean.getBoolean("unstable.test")); // JDK-8196614
-        }
+    public void testSwingNodeMemoryLeak() throws InterruptedException, InvocationTargetException {
         Util.runAndWait(() -> {
             testSwingNodeObjectsInStage();
         });
-        attemptGCSwingNode();
-        assertEquals(TOTAL_SWINGNODE, getCleanedUpSwingNodeCount());
 
-        attemptGCJLabel();
-        assertEquals(TOTAL_SWINGNODE, getCleanedUpJLabelCount());
+        SwingUtilities.invokeAndWait(() -> Util.sleep(500));
```

Comment:
Sleeping on either the JavaFX Application thread or the Swing EDT (as in this case) is not generally desirable. What is the sleep trying to achieve? Is there a better way, possibly sleeping on the test thread and then doing a `SwingUtilities.invokeAndWait` (maybe with a no-op runnable? not sure)?.

Reply:
This is something that stayed in the original patch - I'm no Swing expert so I originally figured I should leave it here. My only guess was that it (among other `Thread.sleep()`-s in this patch) is trying to let the other threads run for a bit and chew through the resources to make sure they're collectable.

Upon closer inspection it looks like JMemoryBuddy is already doing those sleeps, so I'll remove them - tests should work fine without those.